work/v1/types.go

```diff
@@ -189,6 +190,11 @@ const (
 	// If there is conflict, the related Applied condition of manifest will be in the status of False with the
 	// reason of ApplyConflict. This type allows another controller on the spoke to control certain field of the resource.
 	UpdateStrategyTypeServerSideApply UpdateStrategyType = "ServerSideApply"
+
+	// UpdateStrategyTypeReadOnly type means only check the existence of the resource based on the resource's metadata.
```

Comment:
may replace `UpdateStrategyTypeReadOnly` with `ReadOnly` to keep consistent with others.